### PR TITLE
Add url-launcher plugin

### PR DESCRIPTION
**Pull request by szakarias**

### CODE REVIEW

File: packages/url-launcher/example/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        appBar: new AppBar(
+            title: new Text(widget.title),
+        ),
+        body: new Center(
+            child: new Row(
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  new Padding(
+                      padding: new EdgeInsets.all(16.0),
+                      child: new Text("https://flutter.io")),
```
Comment (mit-mit) [inline packages/url-launcher/example/lib/main.dart]: Missing terminating comma

File: packages/url-launcher/lib/url_launcher.dart [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+class UrlLauncher {
+  static const MethodChannel _channel =
+  const MethodChannel('plugins.flutter.io/URLLauncher');
```
Comment (a14n) [inline packages/url-launcher/lib/url_launcher.dart]: nit: indent

File: packages/url-launcher/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# url_launcher
+
+A flutter plugin project for launching a URL.
+
+## Getting Started
```
Comment (szakarias) [inline packages/url-launcher/README.md]: As discussed, I will add this in another PR.

**Comment by szakarias** [comment]
Thanks everyone!

**Pull request merged**